spectrum/input.py

```diff
@@ -108,37 +108,53 @@ def create_blog_article(self, title, text='Lorem ipsum', image=None):
         # TODO: return id and/or node id
 
     def create_article_fragment(self, id, image):
-        create_url = "%s/admin/structure/article_fragment/add" % self._host
-        create_page = self._browser.get(create_url)
-        form = mechanicalsoup.Form(create_page.soup.form)
-        form.input({'name[0][value]': id})
-        form.attach({'files[image_0]': image})
+        filtered_content_url = "%s/admin/content?status=All&type=article&title=%s" % (self._host, id)
+        filtered_content_page = self._browser.get(filtered_content_url)
```

Comment:
I didn't do this originally, but it would be helpful to add `assert filtered_content_page.status_code == 200, "%s returned a %s status code" % (filtered_content_url, filtered_content_page.status_code)` as it is a prerequisite for being able to look for HTML tags later